BRANCHING_RELEASE_STRATEGY.md

```diff
@@ -0,0 +1,178 @@
+# Branching and Release strategy
+
+The following page exposes our branching and release strategy.
+
+## Protected branches
+
+Some branches are protected in our repositories.
+
+Points below detail the policy that determines in our repositories which branches are protected.
+
+### Main branch
+
+The default branch is named `main` and is protected. Commits can be merged only via pull requests passing different checks:
+- Compiling through Ubuntu OS, MacOS and Windows OS without failing
+- Having a testing coverage greater than 80%
```

Comment:
Shouldn't we mention that 90% is asked for powsybl-open-loadflow and powsybl-diagram?